lib/builderator/util.rb

```diff
@@ -79,6 +79,48 @@ def asg(region = Config.aws.region)
         clients["asg-#{region}"] ||= Aws::AutoScaling::Client.new(:region => region)
       end
 
+      def remove_security_group(region = Config.aws.region, group_id = nil)
+        if region.nil?
+          puts "  Dry-run; skipping delete of group_id #{group_id}"
+          return
+        end
+        ec2 = ec2(region)
+        resp = ec2.delete_security_group(group_id: group_id)
```

Comment:
The function would have been called with a nil group_id so it would not have been deleted.  However, this was implicit, and is handled more explicitly now in https://github.com/rapid7/builderator/pull/116/commits/4974bfa7e96dc0c99a70900413e469d6304344f8